src/lib.rs

```diff
@@ -134,6 +135,124 @@ mod util_tests {
     }
 }
 
+/// cec_audio_status which does not allow CEC_AUDIO_VOLUME_STATUS_UNKNOWN
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+pub struct KnownCecAudioStatus(u8);
+
+impl KnownCecAudioStatus {
+    pub fn new(volume: u8, is_muted: bool) -> Self {
+        let volume = min(volume, libcec_sys::CEC_AUDIO_VOLUME_MAX as u8);
+        Self(if is_muted {
+            volume | libcec_sys::CEC_AUDIO_MUTE_STATUS_MASK as u8
+        } else {
+            volume
+        })
+    }
+
+    pub fn volume(self) -> u8 {
+        self.0 & libcec_sys::CEC_AUDIO_VOLUME_STATUS_MASK as u8
+    }
+
+    pub fn is_muted(self) -> bool {
+        self.0 & libcec_sys::CEC_AUDIO_MUTE_STATUS_MASK as u8 != 0
```

Comment:
Would linting break with parentheses?
```suggestion
        (self.0 & libcec_sys::CEC_AUDIO_MUTE_STATUS_MASK as u8) != 0
```